workitem/workitem_repository_blackbox_test.go

```diff
@@ -533,3 +536,151 @@ func (s *workItemRepoBlackBoxTest) TestConcurrentWorkItemCreations() {
+			res, count, err := s.repo.List(context.Background(), fxt.Spaces[0].ID, exp, nil, nil, nil, sort)
+			// then
+			require.NoError(t, err)
+			assert.Equal(t, 7, count)
+			toBeFound := id.Slice{
+				fxt.WorkItems[0].ID,
+				fxt.WorkItems[1].ID,
+				fxt.WorkItems[2].ID,
+				fxt.WorkItems[3].ID,
+				fxt.WorkItems[4].ID,
+				fxt.WorkItems[5].ID,
+				fxt.WorkItems[6].ID,
+			}.ToMap()
+			require.Equal(t, fxt.WorkItems[3].ID, res[6].ID)
+			require.Equal(t, fxt.WorkItems[2].ID, res[5].ID)
+			require.Equal(t, fxt.WorkItems[0].ID, res[4].ID)
+			require.Equal(t, fxt.WorkItems[1].ID, res[3].ID)
+			require.Equal(t, fxt.WorkItems[6].ID, res[2].ID)
+			require.Equal(t, fxt.WorkItems[5].ID, res[1].ID)
+			require.Equal(t, fxt.WorkItems[4].ID, res[0].ID)
+			for _, wi := range res {
+				_, ok := toBeFound[wi.ID]
+				require.True(t, ok, "unknown work item found: %s", wi.ID)
+				delete(toBeFound, wi.ID)
+			}
+			require.Empty(t, toBeFound, "failed to found all work items: %+s", toBeFound)
+		})
+		t.Run("by updated ascending", func(t *testing.T) {
+			// when
+			s.repo.Save(context.Background(), fxt.WorkItems[0].SpaceID, *fxt.WorkItems[3], fxt.Identities[0].ID)
+			s.repo.Save(context.Background(), fxt.WorkItems[0].SpaceID, *fxt.WorkItems[2], fxt.Identities[0].ID)
+			s.repo.Save(context.Background(), fxt.WorkItems[0].SpaceID, *fxt.WorkItems[0], fxt.Identities[0].ID)
+			s.repo.Save(context.Background(), fxt.WorkItems[0].SpaceID, *fxt.WorkItems[1], fxt.Identities[0].ID)
+			s.repo.Save(context.Background(), fxt.WorkItems[0].SpaceID, *fxt.WorkItems[6], fxt.Identities[0].ID)
+			s.repo.Save(context.Background(), fxt.WorkItems[0].SpaceID, *fxt.WorkItems[5], fxt.Identities[0].ID)
+			s.repo.Save(context.Background(), fxt.WorkItems[0].SpaceID, *fxt.WorkItems[4], fxt.Identities[0].ID)
+			exp, _ := query.Parse(ptr.String(`{"system.state": "open"}`))
+			sort, _ := workitem.ParseSortWorkItemsBy(ptr.String("updated"))
+			res, count, err := s.repo.List(context.Background(), fxt.Spaces[0].ID, exp, nil, nil, nil, sort)
+			// then
+			require.NoError(t, err)
+			assert.Equal(t, 7, count)
+			toBeFound := id.Slice{
+				fxt.WorkItems[0].ID,
+				fxt.WorkItems[1].ID,
+				fxt.WorkItems[2].ID,
+				fxt.WorkItems[3].ID,
+				fxt.WorkItems[4].ID,
+				fxt.WorkItems[5].ID,
+				fxt.WorkItems[6].ID,
+			}.ToMap()
+			require.Equal(t, fxt.WorkItems[3].ID, res[0].ID)
+			require.Equal(t, fxt.WorkItems[2].ID, res[1].ID)
+			require.Equal(t, fxt.WorkItems[0].ID, res[2].ID)
+			require.Equal(t, fxt.WorkItems[1].ID, res[3].ID)
+			require.Equal(t, fxt.WorkItems[6].ID, res[4].ID)
+			require.Equal(t, fxt.WorkItems[5].ID, res[5].ID)
+			require.Equal(t, fxt.WorkItems[4].ID, res[6].ID)
+			for _, wi := range res {
+				_, ok := toBeFound[wi.ID]
+				require.True(t, ok, "unknown work item found: %s", wi.ID)
+				delete(toBeFound, wi.ID)
+			}
+			require.Empty(t, toBeFound, "failed to found all work items: %+s", toBeFound)
```

Comment:
grammatical error 
s/found/find

Reply:
Fixed in fc4532d72dbae3555f995aec00b4266cfc524cf2